spring-xsuaa-it/src/test/java/com/sap/cloud/security/xsuaa/extractor/TokenBrokerTestConfiguration.java

```diff
@@ -92,7 +93,7 @@ public OAuth2TokenResponse retrieveAccessTokenViaJwtBearerTokenGrant(URI tokenEn
 					ClientCredentials clientCredentials, String token, @Nullable String subdomain,
 					@Nullable Map<String, String> optionalParameters, boolean disableCacheForRequest)
 					throws OAuth2ServiceException {
-				return null;
+				return new OAuth2TokenResponse(XSUAA_TOKEN, 100, null);
```

Comment:
required?

Reply:
required for TokenBrokerResolverTest when IAS token is provided